lib/client/customer.rb

```diff
@@ -12,7 +12,12 @@ def get(id, params = {})
           return response.body['data'][0]
         rescue Faraday::Error::ClientError => e
           raise BigcommerceApiError.new(
-            'get order', "Failed to get order #{id}", { order_id: id }, e
+            e.response[:status],
+            'get order customer',
+            "Failed to get order customer #{id}",
+            id,
+            { customer_id: id },
```

Comment:
@Jhalter5Stones This is redundant.

The first `id` should be the customer ID (as the identifier). The _data_ object should contain the order ID. And it should also include the errors from the BigCommerce response

Reply:
@mrogers-5s resolved